earth_enterprise/src/common/khstl.h

```diff
@@ -358,6 +358,19 @@ makeset(const T &e1, const T &e2, const T &e3, const T &e4, const T &e5)
   tmp.insert(e5);
   return tmp;
 }
+template <class T>
+std::set<T>
+makeset(const T &e1, const T &e2, const T &e3, const T &e4, const T &e5, const T &e6)
```

Comment:
This is in the KH STL header (`common/khtstl.h`) - I don't want to refactor (and retest every use) of that code.